urdf/robotont_realsense.urdf.xacro

```diff
@@ -1,12 +1,20 @@
 <?xml version="1.0"?>
 <robot xmlns:xacro="http://ros.org/wiki/xacro" name="robotont">
-    <xacro:include filename="$(find realsense2_description)/urdf/_d435.urdf.xacro" />
+	<xacro:include filename="$(find realsense2_description)/urdf/_d435.urdf.xacro" />
     <xacro:include filename="$(find robotont_description)/urdf/robotont_base.urdf.xacro"/>
     <xacro:arg name= "add_plug" default="false"/>
+    <xacro:arg name= "simulated" default="false"/>
     <xacro:property name= "prefix" value="$(arg prefix)"/>
-    <xacro:sensor_d435 parent="${prefix}base_link" use_nominal_extrinsics="false">
-        <origin xyz="-0.12 0 0.21" rpy="0 0 0"/>
-    </xacro:sensor_d435>
+    <xacro:if value="$(arg simulated)">
+        <xacro:sensor_d435 parent="${prefix}base_link" use_nominal_extrinsics="true">
+            <origin xyz="-0.12 0 0.21" rpy="0 0 0"/>
+        </xacro:sensor_d435>
+    </xacro:if>
+    <xacro:unless value="$(arg simulated)">
+        <xacro:sensor_d435 parent="${prefix}base_link" use_nominal_extrinsics="false">
+            <origin xyz="-0.12 0 0.21" rpy="0 0 0"/>
+        </xacro:sensor_d435>
+    </xacro:unless>
```

Comment:
lihtsalt uudishimust küsin, miks simus peab use_nominal_extrinsics olema true ja päriselus mitte?

Reply:
Neil on xacrosse endal pandud selline kommentaar selle kohta: " Use the nominal extrinsics between camera frames if the calibrated extrinsics aren't being published. e.g. running the device in simulation "

Terve see punane osa on ilma läinud gazebo kimpu kasutades ja demodega on kööga:

![nom_true](https://user-images.githubusercontent.com/42968650/96123514-97207900-0efb-11eb-8820-3144b853f987.png)

Reply:
sellest viimasest pildist ei saanud hästi aru. kas punane osa on nüüd kadunud ja demod gazebos katki või on tegelikult praegu kõik hästi?

Reply:
Kui sa praegu oma gazebo kimbu versiooniga proovid, siis ei ole hästi, aga kui ma enda oma ära pushin, siis on.

Reply:
rõõm kuulda.